app/models/concerns/turbo/broadcastable.rb

```diff
@@ -279,6 +283,10 @@ def broadcast_replace(**rendering)
   #   # Sends <turbo-stream action="update" target="clearance_5"><template><div id="clearance_5">Other partial</div></template></turbo-stream>
   #   # to the stream named "identity:2:clearances"
   #   clearance.broadcast_update_to examiner.identity, :clearances, partial: "clearances/other_partial", locals: { a: 1 }
+  #
+  #   # sends <turbo-stream action="update" method="morph" target="clearance_5"><template><div id="clearance_5">Other partial</div></template></turbo-stream>
```

Comment:
Where does the `[method="morph"]` come from in these code samples?

Would the default value of the `:method` attribute be `"replace"`?

Reply:
It's an optional attribute, updated the docs to reflect it.

Reply:
Yes the default value would be replace but we don't need to explicitly pass it as you know! in case the method attribute is not present on the element the action will be processed as replace right?